### ISSUE TITLE
Add a page with accessibility statement

### ISSUE THREAD

**Issue by wfenrich** [CeON/fairchive#863]
To comply with legal requirements, our sites should have an accessibility statement. 

This is not a worldwide requirement, so it is possible that some foreign installation will not have accessibility statement - just like with “Privacy policy”. 

There should be separate text of the statement for each interface language. If there is no statement in a given language, we default to english.

If there is an accessibility statement, the link to “Accessibility statement / Deklaracja dostępności” should be visible in the footer.
